Library/Formula/offzip.rb

```diff
@@ -0,0 +1,15 @@
+class Offzip < Formula
+  version "0.3.6a"
+  homepage "http://aluigi.altervista.org/mytoolz.htm#offzip"
+  url "http://aluigi.altervista.org/mytoolz/offzip.zip"
+  sha256 "71d578d4bcfad52857ef7d2db0b16935ad88aaef493c9b4367d310069efd892b"
+
+  def install
+    system ENV.cc, "offzip.c", "-o", "offzip", "-lz"
+    bin.install "offzip"
+  end
+
+  test do
+    system "offzip|grep #{version}"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!